src/JJMasterData.Core/DataManager/FieldManager.cs

```diff
@@ -34,9 +34,15 @@ public class FieldManager
     public FieldManager(FormElement formElement, ExpressionManager expression)
     {
         FormElement = formElement ?? throw new ArgumentNullException(nameof(formElement));
-        Expression = expression ?? throw new ArgumentNullException(nameof(expression));
+        ExpressionManager = expression ?? throw new ArgumentNullException(nameof(expression));
         Name = "jjpainel_" + formElement.Name.ToLower();
```

Comment:
Because this line is bad for components that use `FieldManager` as a dependency.